pkg/controller/quota/quotaforestmanager/qm_lib_backend_with_quotasubt_mgr.go

```diff
@@ -530,20 +530,19 @@ func (qm *QuotaManager) Fits(aw *arbv1.AppWrapper, awResDemands *clusterstateapi
 	klog.V(4).Infof("[Fits] Sending quota allocation request: %#v ", consumerInfo)
 	allocResponse, err := qm.quotaManagerBackend.AllocateForest(QuotaManagerForestName, consumerID)
```

Comment:
@metalcycling I have implemented fully the changes requested for #398 in this PR.